netci.groovy

```diff
@@ -232,7 +232,7 @@ def branchNames = []
                     batchFile("""set TEMP=%WORKSPACE%\\Binaries\\Temp
 mkdir %TEMP%
 set TMP=%TEMP%
-.\\cibuild.cmd ${(configuration == 'dbg') ? '/debug' : '/release'} ${(buildTarget == 'unit32') ? '/test32' : '/test64'} /buildTimeLimit ${BuildTimeLimit}""")
```

Comment:
The presence of the `/buildTimeLimit` switch on the cibuild.cmd command line is what enables the ProcessWatchdog.